### DISCUSSION TITLE
Sweeper: Handling of no time-sensitive stage one sweeps

### DISCUSSION THREAD

**Discussion by feelancer21** [lightningnetwork/lnd#9170] [Q&A] [answered]
I have the following case at the moment. My node is trying to sweep an outgoing htlc in stage one, which happened because of a self payment. This is not time-sensistive. It started at 2.6 sat/vb and is now at 4.4 sat/vb. I would like to understand how this type of htlcs is handled and which budgets are used for it. Like time-sensistive? Or are they handled in a different way?

https://github.com/lightningnetwork/lnd/tree/master/sweep#sweeping-htlc-outputs

**Comment by ziggie1984** [comment, marked as answer]
So the payment (outgoing htlc) will not be considered when bumping the commitment transaction. So LND will not aggressively bump the fee. Moreover when the commitment is confirmed we are selecting the default deadline of 1008 blocks for the deadline of the outgoing htlc sweep, so it's also the most economic decision we can do, at want point we want to sweep the htlc and get our funds back.

**Comment by feelancer21** [comment]
Thanks Ziggie, 

Does this mean that a LinearFeeFunction is initialized, which then increases over 1008 blocks or is the current value of the estimator simply taken for 1008 blocks and updated if necessary?
Because I have seen bumps in my system, I tend to believe in the first variant. With a max_fee_rate of 100 sat/vb, this results in an increase of 1 sat/vb per 10 blocks. And the bump of 2 sat/vb was within 3 hours.

**Comment by ziggie1984** [comment]
exactly the former, so it starts with a deadline of 1008 blocks and increases the fee via the fee-function after each block is found.
